site/en/docs/privacy-sandbox/attribution-reporting-event-introduction/index.md

```diff
@@ -0,0 +1,321 @@
+
+```text
+ClickID 1298765678762 led to a $317 purchase (red sports shoes) in Boston, CA.
+```
+
+This is supported in [aggregate reports](/docs/privacy-sandbox/attribution-reporting-introduction/#:~:text=Aggregate%20reports%2C%20on%20the%20other%20hand%2C%20offer%20more%20detailed%20conversion%20data).
+{% endAside %}
+
+### One reporting endpoint
+
+Reports can be sent only to **one endpoint**.
+
+{% Aside %}
+More flexible endpoint configurations are [under discussion](https://github.com/WICG/conversion-measurement-api/issues/96).
+{% endAside %}
+
+### Noise
+
+The trigger data (conversion data) is _noised_. Noise is a privacy protection that comes on top of the [data limitation to 3 bits](#detailed-source-side-data-and-limited-conversion-data).
+
+{% Aside "key-term" %}
+When trigger data has its values randomized in some way, this creates _noised data_.
+{% endAside %}
+
+This means that **a small percentage of the time, a report will contain inaccurate trigger data instead of the real data**.
+
+In Chrome, noising works as follows: 5% of the time, inaccurate trigger data will be sent instead of the real trigger data.
+
+#### Example
+
+ClickID `1298765678762` led to a conversion of value `1`.
+
+Instead of receiving this report:
+
+```text
+Click 1298765678762 led to a conversion of value 1.
+```
+
+You may receive the following report:
+
+```text
+Click 1298765678762 led to a conversion of value 4.
+```
+
+### Report limitation
+
+To protect user privacy, the number of reports is limited. For a given source event (click), **at most three conversions can be registered**.
+
+{% Aside  %}
+This means that conversions will be dropped if a user converts more than three times. You can choose which conversions to get and which are fine to drop by configuring [conversion priorities](/docs/privacy-sandbox/attribution-reporting-event-guide/#prioritize-specific-conversions).
+{% endAside %}
+
+#### Example
+
+A user clicks an ad. Later on, they visit the advertiser site (first conversion), then
+create an account (second), and finally make two separate purchases (third and fourth).
+
+Even though they have converted four times, by default, you'll receive a report only for the first three conversions.
+
+### Report scheduling
+
+Reports are **not sent immediately after a user converts**. They're sent with a delay. This is a privacy protection.
+
+After the initial ad click, a schedule of **reporting windows** begins. Each reporting window has a
+deadline. Conversions registered before that deadline will be sent at the end of that window.
+
+{% Aside 'caution' %}
+Reports may not be exactly sent at the scheduled dates. If the browser isn't running
+when a report is scheduled to be sent, the report is sent at browser startup—which could be days or
+weeks after the initially scheduled time.
+{% endAside %}
+
+After expiry (click time + `attributionexpiry`), no additional attribution can be triggered (no additional conversion can be counted).
```

Comment:
Trying spelling out more precisely what you mean in these parenthesis. I think it's the duration of time specified in `attributionexpiry`, starting from the click event?